src/transformers/models/musicgen/modeling_musicgen.py

```diff
@@ -22,13 +22,20 @@
 
 import torch
 import torch.nn as nn
+import torch.nn.functional as F
 from torch.nn import CrossEntropyLoss
 
 from ...activations import ACT2FN
 from ...generation.configuration_utils import GenerationConfig
-from ...generation.logits_process import ClassifierFreeGuidanceLogitsProcessor, LogitsProcessorList
+from ...generation.logits_process import (
+    ClassifierFreeGuidanceLogitsProcessor,
+    LogitsProcessorList,
+)
```

Comment:
Please remove the final comma and put in the same lines !
```suggestion
from ...generation.logits_process import ClassifierFreeGuidanceLogitsProcessor, LogitsProcessorList
```